docs/how-to/authentication-plugin-examples.md

```diff
@@ -11,27 +11,30 @@ Authentication and authorization are crucial components of your Amplication serv
 
 By following this guide and exploring the examples, you'll be well-equipped to implement authentication and authorization in your Amplication-generated service using the authentication plugin that best suits your needs.
 
-## Adding Authentication to Your Service
+## Creating The Authentication Entity
 
-Before adding any specific authentication plugin, you must first enable the Authentication Entity in your service:
+If you initially created your service _without_ authentication, you must first create and define the Authentication Entity. The Authentication Entity **is required** to enable authentication on your service.
```

Comment:
Again- it's relevant to Node.js and not .NET
I think we need to mention it and explain